_data/simple-icons.json

```diff
@@ -6259,6 +6259,11 @@
             "title": "Tata",
             "hex": "486AAE",
             "source": "https://www.tata.com/"
+        },
+		{
+            "title": "TeamCity",
+            "hex": "000000",
+            "source": "https://www.jetbrains.com/teamcity/"
```

Comment:
In my opinion it would be better to use the branding page https://www.jetbrains.com/company/brand/logos as source.

Maybe one of the simple-icons maintainers can help here out.

Reply:
If that is where the SVG and guidelines for TeamCity can be found, then I agree :+1: